pkg/cloud/csvprovider.go

```diff
@@ -299,9 +338,16 @@ func PVValueFromMapField(m string, n *v1.PersistentVolume) string {
 
 func (c *CSVProvider) GetKey(l map[string]string, n *v1.Node) Key {
 	id := NodeValueFromMapField(c.NodeMapField, n, c.UsesRegion)
+	var gpuCount int64
+	gpuCount = 0
+	if gpuc, ok := n.Status.Capacity["nvidia.com/gpu"]; ok { // TODO: support non-nvidia GPUs
```

Comment:
maybe we just something like this now:
kubecost.com/gpu-type

good as-is for my needs though. Thanks Ajay!


just some thoughts- I do think what you have here fits our needs.

Reply:
nm. Looks like this is the suggested label from Kubernetes docs.
https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/

Reply:
feedback from another on-prem customer:
`gpu.nvidia.com/class`

there are no annotations in AWS for the node.

essentially no standard out there.

This customer has thousands of nodes with the above label though. Can we support both?

Reply:
so the two node annotations we have requests for:
```
nvidia.com/gpu_type=Quadro_RTX_4000
gpu.nvidia.com/gpu_type=Quadro_RTX_4000
```

Reply:
Thanks for the feedback. This PR doesn't support multiple yet but it can be done.

Reply:
Should this line be ["nvidia.com/gpu_type"]?

![image](https://user-images.githubusercontent.com/31039225/200835539-75cf0aaa-0fdc-4140-8aa1-4a385a5fb92d.png)

Reply:
No, this is how capacity is counted, not the type of the gpu.